### DISCUSSION TITLE
How to create a .bee_pack file?

### DISCUSSION THREAD

**Discussion by Aether2099** [BEEmod/BEE2-items#4168] [Tech Support / Q&A] [answered]
I'm currently attempting to port over a previous version of a pack I like, and I'm looking into making my own custom assets to be integrated into the interface but I can't seem to figure out how to **open and create** a .bee_pack file. Any help would be greatly appreciated, those are the two main issues I'm facing.

Thanks a ton!

**Comment by Areng14** [comment, marked as answer]
Its just a renamed .zip compress a folder to a zip and change the extension to be a .bee_pack

**Comment by saismeesaimonsaimes** [comment]
You should generally ask questions like this in the Discord server. The GitHub is only for tracking issues.

**Comment by LincoCompany00** [comment]
Does that mean I can take a bee_pack package and change the extension to .zip, and I'll be able to open it?

**Reply by TeamSpen210** [reply]
Yes.

**Reply by LincoCompany00** [reply]
It's not working. I can just rename the file to change the extension, right? I don't have to convert it?

**Reply by TeamSpen210** [reply]
Yes, though builtin Window's zip extractor might not work right. Use 7-Zip.

**Reply by LincoCompany00** [reply]
It'll still work on an android phone, right? That's what I'm using

**Reply by TeamSpen210** [reply]
No idea. Probably should use your computer.

**Reply by LincoCompany00** [reply]
Okay

**Reply by LincoCompany00** [reply]
Uhh... I'm not sure what I did but somwhow I extracted the items just now

**Reply by LincoCompany00** [reply]
Never mind it extracted all the folders without anything in them. I'll use the computer when I get the chance. Thanks for the help!
